test/path-parsing.spec.js

```diff
@@ -1,13 +1,22 @@
 var expect = require("chai").expect;
 var File = require('vinyl');
 var es = require('event-stream');
+var Path = require('path');
 var normalizer = require('../');
 describe("gulp-bower-normalize", function() {
+    var assertFlattened = function(fakeFile, expected) {
+        var myNormalizer = normalizer({flatten: true, bowerJson: './test/fixtures/bower.json'});
+        myNormalizer.write(fakeFile);
+        myNormalizer.once("data", function(file) {
+            expect(file.path).to.equal(Path.normalize(expected));
+        });
+    };
+
     var assertNormalized = function(fakeFile, expected) {
         var myNormalizer = normalizer({bowerJson: './test/fixtures/bower.json'});
         myNormalizer.write(fakeFile);
         myNormalizer.once("data", function(file) {
-            expect(file.path).to.equal(expected);
+            expect(file.path).to.equal(Path.normalize(expected));
```

Comment:
Thanks for getting this in, it's really a pain that the OS world hasn't standardized on this yet. :)